swap_meet/vendor.py

```diff
@@ -1,2 +1,75 @@
+from distutils.command.build_scripts import first_line_re
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+        
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
```

Comment:
This else is not necessary! Since the if block has a return statement, if we move past it it's assumed we're in the else.